setup.cfg

```diff
@@ -39,6 +39,10 @@ where = src
 console_scripts =
     exposurelog = exposurelog.cli:main
 
+[tool:pytest]
+filterwarnings =
+    ignore:numpy.ndarray size changed:RuntimeWarning
```

Comment:
Will you add a note to say why this warning is being ignored?

Reply:
Yes. Good suggestion.